examples/next/docs/js/accessibility/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "a11y-js-demo",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
```

Comment:
Can you add a smoke test for each new examples you added? The other tests are already run within the [code-examples](https://github.com/handsontable/handsontable/blob/develop/.github/workflows/code-examples.yml#L440) workflow. This is a simple smoke test, it can be a copy-pasted from other examples.

Reply:
Adding, thanks for pointing out 😄